lib/basechurch/engine.rb

```diff
@@ -6,6 +6,7 @@
 require 'acts_as_list'
 require 'devise'
 require 'friendly_id'
+require 'validate_url'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.